packages/dom-test-kit/src/stylable-dom-util.ts

```diff
@@ -1,18 +1,23 @@
-import { pseudoStates } from '@stylable/core';
 import {
     parseCssSelector,
     walk,
     stringifySelectorAst,
     SelectorNode,
     Class,
 } from '@tokey/css-selector-parser';
-import type { RuntimeStylesheet, StateValue } from '@stylable/runtime';
+import { type RuntimeStylesheet, type StateValue, statesRuntime } from '@stylable/runtime';
 
 export interface PartialElement {
     querySelector: Element['querySelector'];
     querySelectorAll: Element['querySelectorAll'];
-    className: Element['className'];
-    classList: Element['classList'];
+    className: string;
+    getAttribute: (attr: string) => string | null;
+    classList: {
```

Comment:
inline minimal implemented parts...maybe `Pick<Element['classList'], ...>` is better

Reply:
keeping